.github/jenkinsfile/mainnetBackupPeer.groovy

```diff
@@ -0,0 +1,6 @@
+backupCatapultNodePipeline {
+	agentName = 'mainnet-peer'
+	backupFileName = 'catapult_peer_data.tar.gz'
```

Comment:
would either add `_database` postfix or remove from dual file for naming consistency